animalfacts.py

```diff
@@ -1771,6 +1771,8 @@ def animalfactsbot(reddit):
     'Lobsters have poor eyesight, but have highly developed senses of smell and taste.',
     'Lobsters taste with their legs and chew with their stomachs.',
     'Lobsters keep growing forever. They do not get weaker or lose their ability to reproduce, and will keep on molting and growing.',
+    'Lobsters are invertebrates which means they lack a backbone.'
+    'Lobsters dont have brains as their nervous system is very primitive.'
```

Comment:
Actually, would you mind putting a comma in here so it matches the other tuples?

Reply:
Done! :D

Reply:
Thanks! That means I can fix this new conflict more easily. There were two other lobster facts added and now yours slot right in.